spirv_msl.cpp

```diff
@@ -13679,14 +13679,31 @@ void CompilerMSL::fix_up_shader_inputs_outputs()
 				break;
 			case BuiltInPatchVertices:
 				if (is_tese_shader())
-					entry_func.fixup_hooks_in.push_back([=]() {
-						statement(builtin_type_decl(bi_type), " ", to_expression(var_id), " = ",
-						          to_expression(patch_stage_in_var_id), ".gl_in.size();");
-					});
+				{
+					if (msl_options.raw_buffer_tese_input)
+					{
+						entry_func.fixup_hooks_in.push_back(
+						    [=]() {
+							    statement(builtin_type_decl(bi_type), " ", to_expression(var_id), " = ",
+							              get_entry_point().output_vertices, ";");
```

Comment:
output_vertices? Is that correct? Is it intended that this execution mode is overridden by MoltenVK?

Reply:
I wouldn't mind just adding a set_patch_vertex_inputs() as a separate function to avoid any odd "reinterpretation" of execution modes.

Reply:
Ok.